src/goto-analyzer/goto_analyzer_parse_options.cpp

```diff
@@ -328,19 +494,16 @@ bool goto_analyzer_parse_optionst::set_properties()
     if(cmdline.isset("property"))
       ::set_properties(goto_model, cmdline.get_values("property"));
   }
-
   catch(const char *e)
   {
     error() << e << eom;
     return true;
   }
```

Comment:
Ah, sorry, please ignore my comment. It seems we don't have an exception handler in `doit`.